apps/arena/lib/arena/bots/bot.ex

```diff
@@ -92,6 +92,49 @@ defmodule Arena.Bots.Bot do
   defp maybe_update_state_params(state, game_state) do
     state
     |> Map.put_new(:bot_player_id, get_in(game_state, [:client_to_player_map, state.bot_id]))
+    |> maybe_set_obstacles(game_state)
+  end
+
+  defp maybe_set_obstacles(%{bot_state_machine: %{obstacles: nil}} = state, %{obstacles: obstacles})
```

Comment:
we need to store the obstacles in order to perform collision checks when simplifying the path